packages/back-end/features/AddPersons.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Add persons
+
+  I want a connected user to be able to add a person, either related to another one in their tree or an unrelated one.
+
+  Background: A running app and a user
+    Given a running app
+    And an existing user "{ \"id\": 1, \"name\": \"testuser\", \"email\": \"validemail@tdd.org\", \"password\": \"validpassword1234\"}"
+    And an existing person "{ \"id\": 0, \"name\": \"testuser\"}"
+
+  Scenario: Add a person
+    When I "post" a "{\"name\": \"John Doe\"}" object to "/person" authenticated as user 0
```

Comment:
I think it would be good to decouple this a little:
```suggestion
    Given I am an authentified user
    When I "post" a "{\"name\": \"John Doe\"}" object to "/person" authenticated as user 0
```
Normally the world already has a header field that can be used to do so